base/rhsso/deploy-rhsso.sh

```diff
@@ -4,38 +4,26 @@ set -exuo pipefail
 command -v envsubst
 
 TIMEOUT_TIME=25   # each 5sec: 25 * 5sec = 125sec
-FILE_ROOT=${BASH_SOURCE%/*}
+FILE_ROOT="${BASH_SOURCE%/*}"
 
-NAMESPACE=${NAMESPACE:=tools}
-ADMIN_USERNAME=${ADMIN_USERNAME:="admin"}
-ADMIN_PASSWORD=${ADMIN_PASSWORD:="admin"}
-
-function waitSuccess {
-  TIMEOUT=0
-  CMD=$*
-  until $CMD
-  do
-    if [[ TIMEOUT -eq TIMEOUT_TIME ]]; then
-      echo "Exit due to timeout"
-      exit 1
-    fi
-    TIMEOUT=$((TIMEOUT+1))
-    sleep 5
-  done
-}
+NAMESPACE="${NAMESPACE:=tools}"
+ADMIN_USERNAME="${ADMIN_USERNAME:="admin"}"
+ADMIN_PASSWORD="${ADMIN_PASSWORD:="admin"}"
 
 function deployRHSSO {
-  cat ${FILE_ROOT}/operator-group.yaml.tpl | TARGET_NAMESPACE=${NAMESPACE} envsubst | oc apply -f - -n ${NAMESPACE}
-  oc apply -f "${FILE_ROOT}"/keycloak-subscription.yaml -n ${NAMESPACE}
-  waitSuccess oc wait installplan --all --for=condition=Installed -n ${NAMESPACE}
-
-  cat ${FILE_ROOT}/credential-sso-secret.yaml.tpl | ADMIN_USERNAME=${ADMIN_USERNAME} ADMIN_PASSWORD=${ADMIN_PASSWORD} envsubst | oc apply -f - -n ${NAMESPACE}
-  oc apply -f "${FILE_ROOT}"/sso-keycloak.yaml -n ${NAMESPACE}
-  oc apply -f "${FILE_ROOT}"/no-ssl-sso-service.yaml -n ${NAMESPACE}
-  oc apply -f "${FILE_ROOT}"/no-ssl-sso-route.yaml -n ${NAMESPACE}
-  waitSuccess oc rollout status statefulset/keycloak -w -n ${NAMESPACE}
-
-  oc --namespace ${NAMESPACE} rsh statefulset/keycloak bash -c "/opt/eap/bin/kcadm.sh update realms/master -s sslRequired=NONE --server http://no-ssl-sso:8080/auth --realm master --user ${ADMIN_USERNAME} --password ${ADMIN_PASSWORD} --no-config"
+  TARGET_NAMESPACE="${NAMESPACE}" envsubst < "${FILE_ROOT}"/operator-group.yaml.tpl | oc apply -f - -n "${NAMESPACE}"
+  oc apply -f "${FILE_ROOT}"/keycloak-subscription.yaml -n "${NAMESPACE}"
+  oc wait -n "${NAMESPACE}" --for=jsonpath=status.installPlanRef.name subscription rhsso-operator --timeout=120s
```

Comment:
Is this working for you? I just tried to run this command without any resources and: `jsonpath wait format must be --for=jsonpath='{.status.readyReplicas}'=3`. I would suggest to wait on `status.installPlanRef.kind=InstallPlan` if this error is real

Reply:
Looks like its new format introduced in version 4.15.0

Reply:
You right, time to have it installed from AUR :grimacing: